smartsim/settings/slurmSettings.py

```diff
@@ -243,6 +243,21 @@ def set_broadcast(self, dest_path: t.Optional[str] = None) -> None:
         """
         self.run_args["bcast"] = dest_path
 
+    def set_node_feature(self, feature_list: t.Union[str, t.List[str]]) -> None:
+        """Specify the node feature for this job
+
+        This sets ``--C``
```

Comment:
Only `-C`, i.e. with single dash (don't look at me, it's a Unix convention)

Reply:
oop nice catch